samples/excel/89-preview-apis/style.yaml

```diff
@@ -0,0 +1,241 @@
+        $("#apply-new-style").click(() => tryCatch(applyNewStyle));
+        $("#apply-built-in-style").click(() => tryCatch(applyBuiltInStyle));
+        $("#get-style-font").click(() => tryCatch(getStyleFontProperties));
+        $("#get-style-alignment").click(() => tryCatch(getStyleAlignmentProperties));
+        $("#delete-new-style").click(() => tryCatch(deleteNewStyle));
+
+        async function addNewStyle() {
+            await Excel.run(async (context) => {
+                let styles = context.workbook.styles;
+
+                // Add a new style to the style collection.
+                // Styles is in the Home tab ribbon.
+                styles.add("Diagonal Orientation Style");
+                styles.getItem("Diagonal Orientation Style").orientation = 38;
+                styles.getItem("Diagonal Orientation Style").addIndent = true;
+               
+                let newStyle = styles.getItem("Diagonal Orientation Style");
+                context.load(newStyle);
+
+                newStyle.includeProtection = true;
+                newStyle.shrinkToFit = true;
+                newStyle.locked = false;
+
+                await context.sync();
+
+                OfficeHelpers.UI.notify("Successfully added a new style with diagonal orientation to the Home tab ribbon.");
+            });
+        }
+
+        async function applyNewStyle() {
+            await Excel.run(async (context) => {
+                let worksheet = context.workbook.worksheets.getItem("Sample");
+                context.load(worksheet);
+
+                // Apply new style.
+                worksheet.getRange("A1:E1").style = ("Diagonal Orientation Style");
+                worksheet.getRange("A1:E1").format.verticalAlignment = "Justify";
+
+                let range = worksheet.getRange("A1:E1");
+                context.load(range);
+                context.load(range.format);
+
+                await context.sync();
+            });
+        }
+
+        async function applyBuiltInStyle() {
+            await Excel.run(async (context) => {
+                let worksheet = context.workbook.worksheets.getItem("Sample");
+                context.load(worksheet);
+
+                // Apply built-in style. 
+                // Styles are in the Home tab ribbon.
+                worksheet.getRange("A1:E1").style = Excel.BuiltInStyle.neutral;
+                worksheet.getRange("A1:E1").format.horizontalAlignment = "Right";
+
+                let range = worksheet.getRange("A1:E1");
+                context.load(range);
+                context.load(range.format);
+
+                await context.sync();
+            });
+        }
+
+        async function getStyleFontProperties() {
+            await Excel.run(async (context) => {
+                let style = context.workbook.styles.getItem("Normal");
+                context.load(style);
+                style.font.load("bold, color, italic, name, size");
```

Comment:
I think it would be helpful to developers to know the "forward slash" syntax that they can use to load sub-properties. You can replace all 3 of your loads with just this line:
style.load('font/bold, font/color, font/italic, font/name, font/size, fill/color');